From d3a5065ff9346ea79a77e651a0bebd6a7ce14f6b Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Wed, 30 Dec 2020 20:47:19 +0800
Subject: [PATCH] Revert "arm64: rockchip_defconfig: Disable
 DRM_FBDEV_EMULATION temporarily"

This reverts commit 80cd79aecb9f24a357b940802c91955f712e62e6.

This can work now.

Change-Id: I0650a041dbb00011ac9c292d55e4a848bc193a8a
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 9fcce13bd98f..061597ea9b0d 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -578,7 +578,6 @@ CONFIG_VIDEO_SGM3784=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 CONFIG_DRM_IGNORE_IOTCL_PERMIT=y
-# CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_ROCKCHIP=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
-- 
2.35.3

